tools/actions_local_runner.py

```diff
@@ -346,11 +346,8 @@ def main() -> None:
         relevant_steps = grab_specific_steps(args.step, job)
         future = run_steps(relevant_steps, args.job, relevant_files, quiet)
 
-    if sys.version_info >= (3, 8):
```

Comment:
curious why drop the >= 3.8 constraint? does it mean that we can safely run the loop even with <3.8?

Reply:
Yes, as far as I understand, it wasn't about `loop`, but about https://docs.python.org/3/library/shlex.html#shlex.join